src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/ManagedX509ExtensionProcessor.cs

```diff
@@ -180,7 +180,7 @@ internal static byte[] DecodeX509SubjectKeyIdentifierExtension(byte[] encoded)
         {
             AsnReader reader = new AsnReader(encoded, AsnEncodingRules.BER);
             ReadOnlyMemory<byte> contents;
-            if (!reader.TryReadPrimitiveOctetStringBytes(out contents))
+            if (!reader.TryReadPrimitiveOctetStringBytes(out contents))
```

Comment:
Was this detected as non-ascii whitespace?

Reply:
Yes, it's value 160, a non-breaking space.